io-engine/src/bdev/ftl.rs

```diff
@@ -0,0 +1,258 @@
+//!
+//! The ftl bdev creates a CSAL-FTL disk.
```

Comment:
Can you document how the uri looks like? for example like for https://github.com/openebs/mayastor/blob/6374141862abf68b848e54a5dea2072d6c064819/io-engine/src/bdev/nx.rs#L1